server/conf/messages.en-US

```diff
@@ -46,7 +46,7 @@ content.addressEnteredWithQuestionName={0} entered:
 content.adminLoginPrompt=Looking for something else?
 content.alternativeLoginPrompt=Don''t have an account?
 content.benefits=Get benefits
-content.cannotCheckEligibility=We’re working to fix this issue quickly. Try again at a later time.
```

Comment:
Should we add a test for curly quotes?

Reply:
Not a bad idea - I wrote it down to consider for a future PR